apps/playground/src/app/modules/components/footer-module/footer-module.component.ts

```diff
@@ -0,0 +1,336 @@
+/**
+ * Angular imports
+ */
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import {
+  FormControl,
+  FormGroup,
+  NonNullableFormBuilder,
+} from '@angular/forms';
+import { FooterBrand, FooterCategoryLinks, FooterLink } from '@betagouv/ngx-dsfr/footer';
+
+/**
+ * 3rd-party imports
+ */
+import { Subject, takeUntil } from 'rxjs';
+
+/**
+ * TypeScript entities and constants
+ */
+interface FormFooter {
+  institution: FormControl<string>;
+  description: FormControl<string>;
+  displayBrands: FormControl<boolean>;
+  displayCategoriesLink: FormControl<boolean>;
+  displayOperator: FormControl<boolean>;
+}
+
+const OPERATOR_LOGO: string = 'assets/illustrations/Logo ANCT.svg';
+
+const CATEGORIES_LINKS: FooterCategoryLinks[] = [
```

Comment:
So far, we've been following this convention when it comes to constants : https://github.com/airbnb/javascript#naming--uppercase

Given the fact that we never explicitly talked about it, I'm OK to do so if you have objections with that choice.

If you agree though, please use normal camelCase on lines 28, 30 and 207